docs/source/users/pmodel/pmodel_details/envt_variation_outputs.md

```diff
@@ -38,11 +39,13 @@ co2_1d = np.array([280, 410])
 tc_4d, patm_4d, vpd_4d, co2_4d = np.meshgrid(tc_1d, patm_1d, vpd_1d, co2_1d)
 
 # Calculate the photosynthetic environment
-pmodel_env = pmodel.PModelEnvironment(tc=tc_4d, patm=patm_4d, vpd=vpd_4d, co2=co2_4d)
+pmodel_env = PModelEnvironment(tc=tc_4d, patm=patm_4d, 
+vpd=vpd_4d, 
```

Comment:
strange formatting here and in other places, but maybe not an issue

Reply:
Yup - we should run `jupytext --pipe black` over these files. That applies the `black` autoformatting to the contents of the code cells.

Reply:
@MarionBWeinzierl - I've added a `pre-commit` hook to run this in #247, which also tackles the comment below.